compiler/damlc/tests/daml-test-files/TemplateHeaders.daml

```diff
@@ -23,13 +23,11 @@ template A with
 
 template B a with
     pb : Party
-    xb : a
```

Comment:
This test is primarily for the template headers, so the parameters are not important.